### PR TITLE
Add bitwise blocks to Flow Graph

### PR DESCRIPTION
**Pull request by carolhmj**
Source: https://docs.google.com/spreadsheets/d/17wdC8ML3VzdLLM5t-4HwFcEqnhRgc_8XTuVrF0ynAiI/edit#gid=0

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Review by sebavan — COMMENTED** [review]
Doc is missing and same remark as before on the overall serialization and such. I wonder if you should add all the nodes before having put the full extend of what is needed per node ?

File: packages/dev/core/src/FlowGraph/Blocks/Data/Logic/flowGraphBitwiseBlocks.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    }
+}
+
+export class FlowGraphCountLeadingZerosBlock extends FlowGraphUnaryOperationBlock<number, number> {
+    constructor() {
+        super(RichTypeNumber, RichTypeNumber, (value) => Math.clz32(value));
+    }
+}
+
+export class FlowGraphCountTrailingZerosBlock extends FlowGraphUnaryOperationBlock<number, number> {
```
Comment (RaananW) [inline packages/dev/core/src/FlowGraph/Blocks/Data/Logic/flowGraphBitwiseBlocks.ts:47]: what seb said! :-)

**Review by RaananW — APPROVED** [review]
LGTM. Seb commented - and there is a build issue.

**Comment by RaananW** [comment]
> Doc is missing and same remark as before on the overall serialization and such. I wonder if you should add all the nodes before having put the full extend of what is needed per node ?

We have discussed that offline - it will be implemented in 1-2 weeks.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14334/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14334/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14334/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14334/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14334/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14334/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14334/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14334/merge#BCU1XR#0

**Pull request merged**